addons/mail/static/src/emoji_picker/emoji_picker.js

```diff
@@ -294,7 +306,9 @@ export class EmojiPicker extends Component {
         this.recent[codepoints] ??= 0;
         this.recent[codepoints]++;
         browser.localStorage.setItem("mail.emoji.frequent", JSON.stringify(this.recent));
-        this.gridRef.el.scrollTop = 0;
+        if (this.props.restorePositionOnSelect && !ev.shiftKey) {
+            this.gridRef.el.scrollTop = 0;
+        }
```

Comment:
I don't see the point of `restorePositionOnSelect` prop: why not reset scrollposition in mobile? It feels natural to reset in that context too (Discord does that btw)

Reply:
No, in mobile, when you click once, the EmojiPicker will not reset the scroll position. It only resets when it closes. This behaves differently on mobile and desktop. On the desktop, once we click it, the popover is gone, we can only send one by one. But in mobile, we can do it multiple times, and we don't want to restore the positon after inserting each emoji.

Reply:
Oh I see, when using emoji picker in composer, but not for reactions.
This code should be internally of EmojiPicker. This resets should happen the emoji picker is closed, it should have nothing to do with selection (which is one way to close the emoji picker, but it shouldn't be explicitly coupled with selection. Instead it's the closing behaviour of emoji picker)